micro_ros_diagnostic_bridge/include/micro_ros_diagnostic_bridge/topics.h.in

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+static const char UROS_DIAGNOSTICS_BRIDGE_TOPIC_IN[] = "@DIAGNOSTICS_TOPIC_IN@";
```

Comment:
I think that a more generic `config.hpp.in` would be better, as we may add more configurations. I'm okay with #pragma once but it's against the ROS code style. It should also have a license header.

Do we care if it's not defined? maybe 
#ifndef DIAGNOSTICS_TOPIC_IN
#error DIAGNOSTICS_TOPIC_IN must be defined <or alike>
#endif

Reply:
Not sure about the pre processor magic to check if a variable (`UROS_DIAGNOSTICS_BRIDGE_TOPIC_IN`) is not empty string. But see comment below: I think this should not be completely free anyway, to be consistent with the updater output topic (just allow for an optional prefix, not completely free topic name).